src/js/smooch.jsx

```diff
@@ -196,6 +195,40 @@ export class Smooch {
         });
     }
 
+    monitorUrlChanges() {
+        const onhashchange = window.onhashchange;
+        const pushState = window.history.pushState;
+        const replaceState = window.history.replaceState;
+
+        window.onhashchange = function() {
```

Comment:
I think you can use es6 arrow functions, I believe we use babel in this project, @lemieux ?